examples/SolverComparer.cpp

```diff
@@ -206,10 +200,8 @@ int main(int argc, char *argv[]) {
   }
 
 #ifdef GTSAM_USE_TBB
-  std::unique_ptr<tbb::task_scheduler_init> init;
   if(nThreads > 0) {
     cout << "Using " << nThreads << " threads" << endl;
-    init.reset(new tbb::task_scheduler_init(nThreads));
```

Comment:
Same here. This line was important to the test.

Reply:
Just because we do not explicitly create a `task_scheduler_init` does not mean, TBB is not using its internal one. n threads are still created and the test is run in parallel if TBB is being used, although not needed now, since C++11 has concurrency.

Reply:
If we always wanted max number of threads (i.e. threads = number of cores) that might be ok, but `nThreads` is an optional parameter to this program, so that the user can test timing for a specific number of threads. As I see it, removing this line effectively turns `nThreads` into an on/off switch.

Reply:
Yep, thats right. Thx for the thorough review! I'll add a commit that adds this deprecated feature for now until we figure out how to handle it in the future.